Podfile

```diff
@@ -24,12 +24,9 @@ abstract_target 'Automattic' do
 		pod 'Gridicons', '~> 0.18'
 		pod 'HockeySDK', '5.1.4'
 		pod 'SVProgressHUD', '2.2.5'
-		pod 'Fabric', '1.7.0'
-		pod 'Crashlytics', '3.9'
 
 		# Automattic
-		#
-		pod 'Automattic-Tracks-iOS', '0.3.4'
+		pod 'Automattic-Tracks-iOS', :git => 'https://github.com/Automattic/Automattic-Tracks-iOS.git', :branch => 'add/support-for-tracking-crashes'
 		pod 'Simperium', '0.8.19'
```

Comment:
This caught my eye (although it wasn't modified in your PR). Latest Simperium release was 0.8.21, just by chance... did you base this branch from another (really old) branch?

Reply:
@jleandroperez nice catch — this branch was cut before `0.8.21` was merged into `develop`.

Reply:
Haha we actually just merged in `0.8.21` last week: https://github.com/Automattic/simplenote-ios/commit/662769b49c2ba065f0fe0e4c72a238e4da932e5c.

This branch has been rebased on `develop`, and should be good to go.

One more testing step: please be sure to run `bundle exec fastlane configure_apply` before trying to run this, or it'll fail to find the Sentry DSN and things won't work right :)